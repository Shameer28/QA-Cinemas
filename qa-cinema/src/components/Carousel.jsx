import Carousel from 'react-bootstrap/Carousel'
import image1 from './img/BlackWidow.jpg'
import image2 from './img/Fast9.jpg'
import image3 from './img/JamesBond.jpg'
const Carousels = () => {
    return (
        <Carousel controls={true} fade={true} pause={false}>
            <Carousel.Item interval={2000}>
                <img
                    className="d-block w-100"
                    src={image1}
                    alt="First slide"
                    width="200"
                />

            </Carousel.Item>
            <Carousel.Item interval={2000}>
                <img
                    className="d-block w-100"
                    src={image2}
                    alt="Second slide"
                    width="200"
                />
            </Carousel.Item>
            <Carousel.Item interval={2000}>
                <img
                    className="d-block w-100"
                    src={image3}
                    alt="Third slide"
                    width="200"
                />
            </Carousel.Item>
        </Carousel>
    );
}

export default Carousels;