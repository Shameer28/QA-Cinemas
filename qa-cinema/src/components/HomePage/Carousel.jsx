import Carousel from 'react-bootstrap/Carousel'

import image1 from '../img/MovieThumbnails/BlackWidow.jpg'
import image2 from '../img/MovieThumbnails/F9Landscape.jpg'
import image3 from '../img/MovieThumbnails/JamesBondLandscape.jpg'
import './carousels.css'
const Carousels = () => {
    return (
        <Carousel  controls={false} slide={true} fade={true} pause={false}>
            <Carousel.Item interval={3000}>
                <img
                    className="d-block w-100"
                    src={image1}
                    alt="First slide"
                />
            </Carousel.Item>
            <Carousel.Item interval={3000}>
                <img
                    className="d-block w-100"
                    src={image2}
                    alt="Second slide"
                />               
            </Carousel.Item>
            <Carousel.Item interval={3000}>
                <img
                    className="d-block w-100"
                    src={image3}
                    alt="Third slide"
                />
            </Carousel.Item>
        </Carousel>
    );
}

export default Carousels;