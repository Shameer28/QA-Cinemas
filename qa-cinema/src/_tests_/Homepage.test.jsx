import Homepage from '../components/HomePage/HomePage'
import { render } from "@testing-library/react";
import { create } from 'react-test-renderer';
import Carousel from './../components/HomePage/Carousel'

describe(`Homepage testing`, () => {

    it(`Should match the snapshot`, () => {
        const instance = create(<Homepage/>).toJSON();
        expect(instance).toMatchSnapshot();
    })
})