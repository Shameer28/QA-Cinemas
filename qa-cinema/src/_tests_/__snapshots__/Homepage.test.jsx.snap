// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage testing Should match the snapshot 1`] = `
<div
  class="background"
>
  <div
    className="carousel slide carousel-fade"
    onKeyDown={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
  >
    <ol
      className="carousel-indicators"
    >
      <li
        className="active"
        onClick={[Function]}
      />
      <li
        onClick={[Function]}
      />
      <li
        onClick={[Function]}
      />
    </ol>
    <div
      className="carousel-inner"
    >
      <div
        className="active carousel-item"
        interval={3000}
      >
        <img
          alt="First slide"
          className="d-block w-100"
          src="BlackWidow.jpg"
        />
      </div>
      <div
        className="carousel-item"
        interval={3000}
      >
        <img
          alt="Second slide"
          className="d-block w-100"
          src="F9Landscape.jpg"
        />
      </div>
      <div
        className="carousel-item"
        interval={3000}
      >
        <img
          alt="Third slide"
          className="d-block w-100"
          src="JamesBondLandscape.jpg"
        />
      </div>
    </div>
  </div>
  <div
    className="container"
  >
    <br />
    <h3
      style={
        Object {
          "color": "white",
          "fontWeight": "400",
          "fontfamily": "isonormregular, sans-serif",
          "letterSpacing": ".15em",
          "lineHeight": "1.1",
          "textTransform": "uppercase",
        }
      }
    >
      TOP FILMS
    </h3>
    <br />
    <br />
    <div
      className="row row-cols-md-4"
    >
      <div
        className="col-4"
      >
        <div
          class="image"
        >
          <img
            alt="BlackWidow"
            class="movie"
            height="100%"
            src="BlackWidowPortrait.jpg"
            width="100%"
          />
          <br />
        </div>
        <div
          style={
            Object {
              "color": "white",
              "fontWeight": "400",
              "fontfamily": "isonormregular, sans-serif",
              "letterSpacing": ".15em",
              "lineHeight": "1.1",
              "textAlign": "center",
              "textTransform": "uppercase",
              "width": "75%",
            }
          }
        >
          <br />
          Black Widow
        </div>
        <br />
      </div>
      <div
        className="col-4"
      >
        <div
          class="image"
        >
          <img
            alt="Fast9"
            class="movie"
            height="100%"
            src="Fast9.jpg"
            width="100%"
          />
        </div>
        <div
          style={
            Object {
              "color": "white",
              "fontWeight": "400",
              "fontfamily": "isonormregular, sans-serif",
              "letterSpacing": ".15em",
              "lineHeight": "1.1",
              "textAlign": "center",
              "textTransform": "uppercase",
              "width": "75%",
            }
          }
        >
          <br />
          Fast and Furious 9
        </div>
      </div>
      <div
        className="col"
      >
        <div
          class="image"
        >
          <img
            alt="JamesBond"
            class="movie"
            height="100%"
            src="JamesBond.jpg"
            width="100%"
          />
        </div>
        <div
          style={
            Object {
              "color": "white",
              "fontWeight": "400",
              "fontfamily": "isonormregular, sans-serif",
              "letterSpacing": ".15em",
              "lineHeight": "1.1",
              "textAlign": "center",
              "textTransform": "uppercase",
              "width": "100%",
            }
          }
        >
          <br />
          James Bond No Time To Die
        </div>
      </div>
    </div>
    <div
      className="row row-cols-md-4"
    >
      <div
        className="col-4"
      >
        <div
          class="image"
        >
          <img
            alt="AQuietPlace2"
            class="movie"
            height="100%"
            src="AQuietPlace2.jpg"
            width="100%"
          />
        </div>
        <div
          style={
            Object {
              "color": "white",
              "fontWeight": "400",
              "fontfamily": "isonormregular, sans-serif",
              "letterSpacing": ".15em",
              "lineHeight": "1.1",
              "textAlign": "center",
              "textTransform": "uppercase",
              "width": "75%",
            }
          }
        >
          <br />
          A Quiet Place II
        </div>
        <br />
      </div>
      <div
        className="col-4"
      >
        <div
          class="image"
        >
          <img
            alt="Conjuring"
            class="movie"
            height="100%"
            src="Conjuring.jpg"
            width="100%"
          />
        </div>
        <div
          style={
            Object {
              "color": "white",
              "fontWeight": "400",
              "fontfamily": "isonormregular, sans-serif",
              "letterSpacing": ".15em",
              "lineHeight": "1.1",
              "textAlign": "center",
              "textTransform": "uppercase",
              "width": "75%",
            }
          }
        >
          <br />
          Conjuring: Devil made me do it
        </div>
      </div>
      <div
        className="col"
      >
        <div
          class="image"
        >
          <img
            alt="Nobody"
            class="movie"
            height="100%"
            src="Nobody.jpg"
            width="100%"
          />
        </div>
        <div
          style={
            Object {
              "color": "white",
              "fontWeight": "400",
              "fontfamily": "isonormregular, sans-serif",
              "letterSpacing": ".15em",
              "lineHeight": "1.1",
              "textAlign": "center",
              "textTransform": "uppercase",
              "width": "100%",
            }
          }
        >
          <br />
          Nobody
        </div>
      </div>
    </div>
  </div>
</div>
`;
